src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -108,7 +110,9 @@ public void HandleParentProcessExit()
             this.Dispose();
 
             EqtTrace.Info("DesignModeClient: Parent process exited, Exiting myself..");
-            Environment.Exit(1);
+
+            // ToDo: Initialize this in .ctor, so that it is mockable
+            new PlatformEnvironment().ExitCurrentProcess(1);
```

Comment:
I think the API could be called `Exit`, it doesn't seem appropriate for Environment to ever expose an API to exit another process.